docs/en/essentials/passing-props.md

```diff
@@ -59,7 +58,7 @@ const router = new VueRouter({
 
 ### Function mode
 
-You can create a function that returns props. This allows you to cast the parameter to another type, combine static values with route-based values, etc.
+You can create a function that returns props. This allows you to coerce the parameter to another type, combine static values with route-based values, etc.
```

Comment:
I'm not sure to well understand difference between « cast » and « coerce ».

For me, « cast » is the action to explicitly convert a type value into another with some keyword usage.
And « coerce » is the action to implicitly convert a type value with the usage of a context or operand.

You means « with the usage of operand » is not a coercing but a casting too? Same with the conversion is « implicite » (no « int » or « string » keyword for example)?